elisp-bytecode.texi

```diff
@@ -1224,76 +1323,128 @@ Call @code{current-buffer} with no arguments.
 @kindex set-buffer
 Call @code{set-buffer} with one argument.
 
+@node save-current-buffer-1
+@unnumberedsubsubsec @code{save-current-buffer-1} (114)
+@kindex save-current-buffer
+Call @code{save-current-buffer} with one stack argument,
```

Comment:
There is no stack argument, as far as I can see.

Reply:
Thanks. I was wondering what was going on in the bytecode.